src/kOS/Module/kOSVesselModule.cs

```diff
@@ -394,6 +394,11 @@ private void CheckRehookAutopilot()
         /// <param name="c"></param>
         private void UpdateAutopilot(FlightCtrlState c)
         {
+            // Lock out controls if insufficient avionics in RP-0.
+            ControlTypes RP0Lock = InputLockManager.GetControlLock("RP0ControlLocker");
```

Comment:
Question:  Is this RP0 lock still called RP0Lock in RP-1?  And if so, does that constitute a promise from the RP folks that this will be the public API call name of it for other mods to check for through all future versions?  I suspect it's only called ``RP0Lock`` purely because nobody noticed the name still said "RP0" when RP-1 came out.  That sounds like the sort of thing that might get fixed when someone notices it, and then that breaks mod compatibility for a mere name change.  Do you know if this is going to be the permanent name for this?

Reply:
The name is taken directly from the current rp-1 source.
That said it's entirely possible someone could unknowingly change it in the future. I might restructure the way this works so that kOS provides a function for rp-1 to call to lock controls. That way it is obvious that there is an external interaction.

Reply:
I got a reply in the RP-0 Github about this - they claim it will remain "RP0" forever and it's safe to use this name.